### PR TITLE
New Settings -> Graphics... dialog for Classic UI users

### PR DESCRIPTION
**Pull request by zfox23**
[BUGZ-814](https://highfidelity.atlassian.net/browse/BUGZ-814)

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/20345/HighFidelity-Beta-PR15918-4c7d0fc.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/20345/HighFidelity-Beta-Interface-PR15918-4c7d0fc.dmg

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/20345/HighFidelity-Beta-PR15918-4c7d0fc.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/20345/HighFidelity-Quest-Beta-PR15918-4c7d0fc.apk)

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/20345/HighFidelity-Beta-PR15918-4c7d0fc.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/20345/HighFidelity-Beta-Interface-PR15918-4c7d0fc.exe

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/20349/HighFidelity-Beta-PR15918-56a55ef.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/20349/HighFidelity-Quest-Beta-PR15918-56a55ef.apk)

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/20349/HighFidelity-Beta-PR15918-56a55ef.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/20349/HighFidelity-Beta-Interface-PR15918-56a55ef.dmg

File: interface/resources/qml/hifi/dialogs/graphics/GraphicsSettings.qml [hunk 1]
```diff
@@ -0,0 +1,379 @@
...
+                        id: resolutionScaleSlider
+                        enabled: performanceCustom.checked
+                        anchors.left: resolutionHeader.right
+                        anchors.leftMargin: 57
+                        anchors.top: parent.top
+                        width: 150 
+                        height: parent.height
+                        colorScheme: hifi.colorSchemes.dark
+                        minimumValue: 0.25
+                        maximumValue: 1.5
```
Comment (samcake) [inline interface/resources/qml/hifi/dialogs/graphics/GraphicsSettings.qml]: I wouldn;t go up to 1.5
but just max at 1.0 instead

Comment (zfox23) [reply interface/resources/qml/hifi/dialogs/graphics/GraphicsSettings.qml]: Do we not want to enable supersampling for power users?

Comment (zfox23) [reply interface/resources/qml/hifi/dialogs/graphics/GraphicsSettings.qml]: OK, I changed this to 1.0.

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/20365/HighFidelity-Beta-PR15918-e039264.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/20365/HighFidelity-Quest-Beta-PR15918-e039264.apk)

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/20365/HighFidelity-Beta-PR15918-e039264.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/20365/HighFidelity-Beta-Interface-PR15918-e039264.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/20365/HighFidelity-Beta-PR15918-e039264.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/20365/HighFidelity-Beta-Interface-PR15918-e039264.exe

File: interface/resources/qml/hifi/dialogs/graphics/GraphicsSettings.qml [hunk 2]
```diff
@@ -0,0 +1,379 @@
...
+        id: graphicsSettingsColumnLayout
+        anchors.left: parent.left
+        anchors.leftMargin: 26
+        anchors.right: parent.right
+        anchors.rightMargin: 26
+        anchors.top: parent.top
+        spacing: 8
+
+        ColumnLayout {
+            id: avatarNameTagsContainer
```
[the remark this answers is not in the document]
Comment (zfox23) [reply interface/resources/qml/hifi/dialogs/graphics/GraphicsSettings.qml]: NOPE, great catch, thank you :)

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/20370/HighFidelity-Beta-PR15918-5828f46.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/20370/HighFidelity-Quest-Beta-PR15918-5828f46.apk)

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/20370/HighFidelity-Beta-PR15918-5828f46.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/20370/HighFidelity-Beta-Interface-PR15918-5828f46.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/20370/HighFidelity-Beta-PR15918-5828f46.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/20370/HighFidelity-Beta-Interface-PR15918-5828f46.exe

**Pull request merged**